src/deluge/storage/flash_storage.cpp

```diff
@@ -291,6 +294,8 @@ void resetSettings() {
 	resetAutomationSettings();
 
 	defaultStartupSongMode = StartupSongMode::BLANK;
+
+	defaultSliceMode = SampleRepeatMode::ONCE;
```

Comment:
Isn't this changing the default Deluge behavior? Shouldn't this be CUT so if the users don't ever touch the Settings, it will behave like always? @seangoodvibes